src/services/TestResultsService.ts

```diff
@@ -270,34 +279,49 @@ export class TestResultsService {
       return validationErrors;
     }
     for (const testType of testResult.testTypes) {
-      const context = {isPassed: testType.testResult, isSubmitted: testResult.testStatus};
+      const options = {abortEarly: false, context: {isPassed: testType.testResult, isSubmitted: testResult.testStatus}};
```

Comment:
can we rename `isPassed` to `hasTestResult` and `isSubmitted ` to `hasStatus`

Reply:
renamed isPassed and removed isSubmitted as it was not used anymore